### PR TITLE
feat(icons): add new icons in launchpad

### PR DESCRIPTION
**Pull request by tiffanylphan**
## Summary

Add AI Config icons 🐌 
https://www.figma.com/design/eSqxBJ2xm4wKVTVEBzTr5T/AI-Configs?node-id=87-8745&node-type=canvas&t=LU1KaBg6KplKqPhY-0

### CODE REVIEW

**Review by bjlagman12 — APPROVED** [review]

**Review by Niznikr — CHANGES_REQUESTED** [review]
You'll need to get rid of any fills.

File: packages/icons/src/types.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ const icons = [
 	'analyze',
 	'animal-cat',
 	'animal-turtle',
+	'anthropic',
```
Comment (Niznikr) [inline packages/icons/src/types.ts:10]: @tiffanylphan can you update this list?

File: packages/icons/src/img/sprite.svg [hunk 2]
```diff
@@ -55,7 +55,6 @@
         d="m6.135 6.585 1.693 1.832h2.677L12.2 6.585a5.886 5.886 0 0 0-3.032-.835 5.886 5.886 0 0 0-3.032.835Zm7.247.93-1.597 1.727 1.117 1.675h2.134a5.868 5.868 0 0 0-.744-2.208 5.94 5.94 0 0 0-.91-1.194Zm3.164 3.402H17a.583.583 0 0 0 0-1.167h-.667c.101.378.173.768.213 1.167ZM15.75 8.25a7.449 7.449 0 0 0-1.949-2.373A7.388 7.388 0 0 0 9.168 4.25c-1.753 0-3.366.61-4.636 1.627a7.407 7.407 0 0 0-2.743 5.04h-.121a.75.75 0 0 0 0 1.5h1.686l.404 2.694a.75.75 0 0 0 .742.639H5a.75.75 0 0 0 .728-.568l.691-2.765h5.496l.69 2.765a.75.75 0 0 0 .728.568H14a.75.75 0 0 0 .739-.62l.474-2.713H17a2.083 2.083 0 0 0 0-4.167h-1.249Zm-4.65 2.667-.667-1h-2.53l-.667 1H11.1Zm-5.667 0 1.116-1.675-1.596-1.727a5.896 5.896 0 0 0-1.655 3.402h2.135Z"
         clip-rule="evenodd"
       />
-    </symbol>
```
Comment (Niznikr) [inline packages/icons/src/img/sprite.svg:58]: @tiffanylphan this removal seems unexpected

File: packages/icons/src/types.ts (hunk 1 above)
Comment (tiffanylphan) [reply packages/icons/src/types.ts:10]: Yeah I'll let you know when it's ready again, sorry I'm in between things

Comment (Niznikr) [reply packages/icons/src/types.ts:10]: No rush!

File: packages/icons/src/types.ts [hunk 3]
```diff
@@ -256,17 +259,21 @@ const icons = [
...
+	'thumb-down-outline',
 	'thumb-up',
+	'thumb-up-outline',
 	'timeline-text',
 	'toggle-bolt',
 	'toggle-off',
 	'toggle-on',
 	'toggle-stack',
 	'tools',
+	'topology-star-3',
```
Comment (tiffanylphan) [inline packages/icons/src/types.ts:276]:
```suggestion
```

**Review by Niznikr — CHANGES_REQUESTED** [review]
Still seeing fills defined. Also the sprite file has more added than what the types diff shows.

**Review by Niznikr — CHANGES_REQUESTED** [review]
Almost there! Not seeing markdown off render

Screenshot 2024-10-16 at 1 42 42 PM

**Review by Niznikr — APPROVED** [review]

**Pull request merged**